app/views/arts/index.html.erb

```diff
@@ -1,23 +1,32 @@
-<div class="arts-container">
-<div class="row">
-    <% @arts.each do |art| %>
-      <div class="col-sm-3 center-card">
-        <%= link_to art_path(art), class: "art-card" do %>
-          <div class="art-card-img" style="background-image: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)),
-            url('<%= cl_image_path art.photos.first.key %>');">
-            <div class="favorites-location-div">
-              <i class="fa-solid fa-heart favorites-gold"></i>
-            </div>
-          </div>
-          <div class="art-card-body">
-            <h5 class="art-card-title"><%= art.title %></h5>
-            <div>
-              <p class="art-card-text"><%= art.description %></p>
-              <p class="art_card-price">€ <%= art.price.round(2) %> per day</p>
-            </div>
-          </div>
-        <% end %>
+<div class="container-default category-container">
+  <div class="category-btn category-left-btn">
+  <i class="fa-solid fa-arrow-left cat-circle"></i>
+  </div>
+    <% @categories.each do |c| %>
+      <div class="category-card category-card-img" style="background-image: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url('<%= cl_image_path c.photo.key %>');">
+        <p class="category-card-text"><%= c.name %></p>
       </div>
     <% end %>
+  <div class="category-btn category-right-btn">
+  <i class="fa-solid fa-arrow-right cat-circle"></i>
+  </div>
 </div>
+
+<div class="arts-container container-default">
+  <% @arts.each do |art| %>
+    <%= link_to art_path(art), class: "art-card" do %>
+      <div class="art-card-img" style="background-image: linear-gradient(rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), url('<%= cl_image_path art.photos.first.key %>');">
+        <div class="favorites-location-div">
+          <i class="fa-solid fa-heart favorites-gold"></i>
+        </div>
+      </div>
+      <div class="art-card-body">
+        <h5 class="art-card-title"><%= art.title %></h5>
+        <div>
+          <p class="art-card-text"><%= art.description %></p>
+          <p class="art_card-price">€ <%= art.price.round(2) %> per day</p>
+        </div>
+      </div>
+    <% end %>
+  <% end %>
 </div>
```

Comment:
Can you give this a bit of spacing between siblings to make it more readable ?

And can you wrap the whole document into a div with an id of arts-index please ?